zxlive/rule_panel.py

```diff
@@ -80,3 +80,7 @@ def get_rule(self) -> CustomRule:
                           self.graph_scene_right.g,
                           self.name_field.text(),
                           self.description_field.text())
+    
+    def shutdown(self):
```

Comment:
Why do you need this method here in `rule_panel` if it is already implemented in `base_panel`?